### ISSUE TITLE
Pandas SettingWithCopyWarning

### ISSUE THREAD

**Issue by dweindl** [ICB-DCM/PEtab#91]
Introduced in eae08e2146275dc95f7b30c75389f0000fbd70d2. 
@DantongWang  Can you please fix that?

```
tests/test_visualization.py::test_generate_experimentId_no_empty
tests/test_visualization.py::test_generate_experiment_id_empty
  /home/dweindl/src/PEtab/petab/generate_experiment_id.py:80: SettingWithCopyWarning: 
  A value is trying to be set on a copy of a slice from a DataFrame
  
  See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/indexing.html#indexing-view-versus-copy
    measurement_data.experimentId[ind] = 'experiment_' + str(count)

tests/test_visualization.py::test_generate_experimentId_no_empty
tests/test_visualization.py::test_generate_experiment_id_empty
  /home/dweindl/.local/lib/python3.7/site-packages/pandas/core/indexing.py:190: SettingWithCopyWarning: 
  A value is trying to be set on a copy of a slice from a DataFrame
  
  See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/indexing.html#indexing-view-versus-copy
    self._setitem_with_indexer(indexer, value)

-- Docs: https://docs.pytest.org/en/latest/warnings.html
```

**Comment by DantongWang** [comment]
Problem solved now. Should I merge it to 'develop' or 'master' branch?

**Comment by dweindl** [comment]
Great. PR to develop please.

**Comment by dweindl** [comment]
Closed by #97

**Comment by dweindl** [comment]
Fixed in 0.0.0a14
